src/components/MovieCard/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  WrapItem,
+  Link,
+  Image,
+  Heading,
+  Text,
+  List,
+  ListItem,
+} from '@chakra-ui/react';
+import CardImage from '../../assets/card-image.png';
+import StarFull from '../../assets/starFull.png';
+import StarEmpty from '../../assets/starEmpty.png';
+export const MovieCard = (): JSX.Element => {
+  return (
+    <WrapItem
+      bg="white"
+      p="2.4rem"
+      maxW="343px"
+      borderTopLeftRadius="10rem"
+      borderTopRightRadius="1.6rem"
+      borderBottomLeftRadius="1.6rem"
+      borderBottomRightRadius="10rem"
+    >
+      <Link display="block" _hover={{ textDecor: 'none' }}>
+        <Image src={CardImage} maxW="295px" />
+        <Heading
+          color="dark.900"
+          fontSize="3.6rem"
+          fontWeight="400"
+          lineHeight="4.19rem"
+          letterSpacing="0.18em"
+          fontFamily="Roboto"
+          mt="1.3rem"
+          whiteSpace="nowrap"
+        >
+          Home Alone 2
+        </Heading>
+        <Text fontSize="1.8rem" letterSpacing="0.16em">
+          Comedy
+        </Text>
+        <List display="flex">
+          <ListItem>
```

Comment:
bro there is another way to render it for example:
```typescript
import { useMemo } from 'react'

import StarEmpty from '../../assets/starEmpty.png'
import StarFull from '../../assets/starFull.png'

interface IStars {
  rating: number
}

const Stars = ({ rating }: IStars) => {
  const maxRating = 100
  const maxStars = 5

  const starsWillFilled = useMemo(() => Math.round((maxStars * rating) / maxRating), [maxStars, rating, maxRating])
  const stars = new Array(maxStars).fill('')

  return (
    <List display='flex'>
      {stars.map((_, index) =>
        index < starsWillFilled ? (
          <ListItem>
            <Image src={StarFull} alt='start filled' />
          </ListItem>
        ) : (
          <ListItem>
            <Image src={StarEmpty} alt='start unfilled' />
          </ListItem>
        ),
      )}
    </List>
  )
}

```

Reply:
I will use it when fetching data dynamically